README.adoc

```diff
@@ -0,0 +1,274 @@
+= Atomic Developer Bundle
+:toc:
+
```

Comment:
+1

BTW, don't you guys have a setup to let you edit and view the changes locally? It is a bit tedious to have to push to GitHub to see changes rendered.